test/util.js

```diff
@@ -250,6 +250,16 @@ export function touchmove(el, x, y, bbEl) {
     changedTouches: [{ x: 0, y: 0 }],
   });
 }
+export function handlemove(grid, xDiff, yDiff) {
```

Comment:
This is the cause of the failing test. This should move the handle, but somehow `mousedown` doesn't receive `self.dragMode`. I think the initial `mousemove` should resolve that, but it doesn't. Do you know any other reliable way to fix this?

Reply:
Turns out the handle wasn't visible, so mouse events were failing. Now the test should pass.